bench-tps/src/cli.rs

```diff
@@ -332,10 +338,18 @@ pub fn build_args<'a>(version: &'_ str) -> App<'a, '_> {
                 .help("Controls the connection pool size per remote address; only affects ThinClient (default) \
                     or TpuClient sends"),
         )
+        .arg(
+            Arg::with_name("compute_unit_price")
+            .long("compute-unit-price")
+            .takes_value(true)
+            .validator(|s| is_within_range(s, 0..))
```

Comment:
Just to confirm, is the unit for this parameter "micro-lamports"?

Reply:
yes, micro-lamports. it's just directly pushed into the compute budget instruction

Reply:
Got it. As a nit, maybe add this to `.help("Sets constant compute-unit-price in micro-lamports to ...` to clarify? In case priority fee eventually gets to round to milli-lamports. Or perhaps better to ask user enter milli-lamports in cli, then multiple `COMPUTE_UNIT_PRICE_MULTIPLIER` in code.